app-ktor-server/src/main/resources/application.conf

```diff
@@ -48,7 +48,7 @@ ktor {
         # to download the app
         androidDownloadHref = "https://play.google.com/store/apps/details?id=com.toughra.ustadmobile"
 
-
+        useMockEmail = "true"
```

Comment:
This must be commented out. It should be false by default. It will only be enabled when running in a test using the testserver controller.

It should have comments explaining its purpose.